scripts/photo.js

```diff
@@ -73,6 +73,34 @@ photo.load = function(photoID, albumID) {
 
 };
 
+photo.update_overlay_type = function() {
+	// Only run if the overlay is showing
+	if(!lychee.image_overlay)
+	{
+		return false;
+	}
+	else
+	{
+		console.log('Current ' + lychee.image_overlay_type);
+		let types = ['exif', 'desc', 'takedate'];
+
+		let i = types.indexOf(lychee.image_overlay_type);
+
+		if((i+1) > types.length - 1)
+		{
+			lychee.image_overlay_type = types[0];
+			$('#image_overlay').remove();
+			lychee.imageview.append(build.overlay_image(photo.json))
+		}
+		else
+		{
+			lychee.image_overlay_type = types[i+1];
+			$('#image_overlay').remove();
+			lychee.imageview.append(build.overlay_image(photo.json))
+		}
```

Comment:
you can do it simpler, instead of a `if` you can use the modulo: `%`:
```js
lychee.image_overlay_type = types[(i+1)%types.length];
$('#image_overlay').remove();
lychee.imageview.append(build.overlay_image(photo.json))
```